lighthouse-core/gather/driver.js

```diff
@@ -25,13 +25,15 @@ const URL = require('../lib/url-shim');
 const log = require('../lib/log.js');
 const DevtoolsLog = require('./devtools-log');
 
-const PAUSE_AFTER_LOAD = 5000;
+const DEFAULT_PAUSE_AFTER_LOAD = 0;
+const DEFAULT_NETWORK_QUIET_THRESHOLD = 5000;
+const DEFAULT_PAUSE_AFTER_NETWORK_QUIET = 0;
 
 const _uniq = arr => Array.from(new Set(arr));
 
 class Driver {
   static get MAX_WAIT_FOR_FULLY_LOADED() {
-    return 25 * 1000;
+    return 30 * 1000;
```

Comment:
increased to account for the fact that the trace will end abruptly after this time instead of 5s after it, so we still aren't capturing any larger traces than 30s